static/app/views/issueList/overview.spec.tsx

```diff
@@ -57,12 +57,19 @@ const routerProps = {
 describe('IssueList', function () {
   let props;
```

Comment:
this is typed as any and why this test didn't take much to convert

Reply:
should we fix it 😬